src/views/screenShareView.jsx

```diff
@@ -66,7 +65,6 @@ export const ScreenShareView = ({ isChatOpen, toggleChat }) => {
 export const SidePane = ({
   isChatOpen,
   toggleChat,
-  isPresenterInSmallTiles,
   peerScreenSharing, // the peer who is screensharing
```

Comment:
why is this "isPresenterInSmallTiles" removed?

Reply:
There are cases where the presenter is already present in small tiles and you don't want to show him again.